.github/stale.yml

```diff
@@ -10,17 +10,17 @@ pulls:
   daysUntilStale: 7
   # Number of days of inactivity before a stale Issue or Pull Request is closed.
   # Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
-  daysUntilClose: 7
+  daysUntilClose: 30
```

Comment:
I think it is fine to close after 7 days after an issue/pr is declared state.

We should however bump up daysUntilStale from 7 to maybe 14 for PRs.

Reply:
I don't think 1 week after warning to close is enough